### DISCUSSION TITLE
Is it possible to prevent modal closing after form submits?

### DISCUSSION THREAD

**Discussion by luke-parallax** [filamentphp/filament#10967] [Help]
Hey,

I've leveraged the action modal slide-over feature to create a comments system within my dashboard. The only thing left to do is prevent the modal from closing after the form submits. Is there an easy way to do this? So far I've had no luck in doing so.

I'd appreciate any suggestions!

Thanks!

**Comment by leandrocfe** [comment]
`this->halt()` should work. Can you share the code please?
